File: Competitor Answers/Hüseyin Bora İÇÖZ - Ramazan Semih UĞUR/soru2.py
dizi1 = [8, 10, 19, 6, 17, 14, 15, 2]
dizi2 = [17, 18, 11, 2, 10, 4, 15, 3]
dizi3 = [11, 2, 9, 19, 20, 10 , 12, 14]

print("Dizi 1:" + str(dizi1))
print("Dizi 2:" + str(dizi2))
print("Dizi 3:" + str(dizi3))

def diziUzunlugu(dizi):
	sayac1 = 0
	for v in dizi:
		sayac1 = sayac1 +1
	return sayac1

def diziyiSirala(dizi):
	baslangic = 0
	for i in range(0,diziUzunlugu(dizi)):
		for j in range(baslangic, diziUzunlugu(dizi)):
			if j!=diziUzunlugu(dizi)-1:
				if dizi[i]>dizi[j+1]:
					temp = dizi[j+1]
					dizi[j+1] = dizi[i]
					dizi[i] = temp
		baslangic = baslangic + 1

def dizileriBirlestir(ilkDizi, ikinciDizi):
	for i in range(0,diziUzunlugu(ilkDizi)):
		tekrar = False
		for val in ikinciDizi:
			if ilkDizi[i] == val:
				tekrar = True
		if tekrar == True:
			continue
		elif tekrar == False:
			ikinciDizi.append(ilkDizi[i])

dizileriBirlestir(dizi1,dizi3)
diziyiSirala(dizi3)
dizileriBirlestir(dizi2,dizi3)
diziyiSirala(dizi3)

print("Dizilerin birlestirilmis ve siralanmis hali: "+ str(dizi3))

File: Yarışma Cevapları/Berk Sedat KABADAYI - Ali İHSAN/Soru 2.js
let dizi1 = [8,10,19,6,17,14,15,2]
let dizi2 = [17,18,11,2,10,4,15,3]
let dizi3 = [11,2,9,19,20,10,12,14]

dizi2.forEach(a => {
    dizi1.push(a)
})

dizi3.forEach(a => {
    dizi1.push(a)
})

let temp = 0;

for (let i=0; i < dizi1.length; i++) {
    for (let j=0; j < dizi1.length - i - 1; j++) {
        if (dizi1[j + 1] < dizi1[j]) {
            temp = dizi1[j]
            dizi1[j] = dizi1[j + 1]
            dizi1[j + 1] = temp
        }
    }
}

for (let i = 0; i< dizi1.length; i++) {
    for (let j = 0; j < dizi1.length; j++) {
        if (dizi1[i] == dizi1[j] && i != j) dizi1.splice(j,1)
    }
}

console.log(dizi1)